apps/prairielearn/src/lib/config.ts

```diff
@@ -70,6 +70,7 @@ const ConfigSchema = z.object({
   sessionStoreExpireSeconds: z.number().default(86400),
   sessionCookieNames: z.array(z.string()).default(['prairielearn_session']),
   sessionCookieSameSite: z.string().default(process.env.NODE_ENV === 'production' ? 'none' : 'lax'),
+  cookieDomain: z.string().optional(),
```

Comment:
TODO: make this non-optional in production environment to force self-hosters to provide a domain so that all our cookie migration stuff will work as expected.

Reply:
Done in bee055d.